front/src/config/i18n/fr.json

```diff
@@ -1577,6 +1577,9 @@
       },
       "system": {
         "start": "Gladys démarre"
+      },
+      "message-queue": {
+        "received": "Recevoir un message de la file d'attente"
```

Comment:
What is the logic behind avoiding to talk about MQTT everywhere in this feature? :) 

Is the goal to be able to handle multiple types of queues?

Because right now it feels confusing, if a user wants to use MQTT here, he'll be really confused